java/test/jmri/jmrix/can/cbus/swing/nodeconfig/CbusAllocateNodeNumberTest.java

```diff
@@ -30,6 +31,7 @@
 public class CbusAllocateNodeNumberTest {
 
     @Test
+    @DisabledIfSystemProperty(named ="java.awt.headless", matches ="true")
```

Comment:
Is the node number allocation code being tested inside of a GUI object?  I would think this process can be completed completely without a GUI object, so you shouldn't need the headless check.

Reply:
Yep this check can be removed.